frontend/components/RoleChooser.tsx

```
@@ -87,37 +93,37 @@ export default function RoleChooser({
           {/* Professor Card */}
           <div 
-            className={`cursor-pointer p-6 rounded-lg border-2 transition-all duration-200 ${
+            className={`card-elevated cursor-pointer p-8 rounded-xl border-2 transition-all duration-300 backdrop-blur-sm ${
               selectedRole === "professor" 
-                ? "border-purple-500 bg-purple-900/20" 
-                : "border-gray-600 bg-gray-900/20 hover:border-gray-500"
+                ? "border-purple-500 bg-gradient-to-br from-purple-900/40 to-purple-800/20 shadow-lg shadow-purple-500/20 scale-105" 
+                : "border-gray-600/60 bg-gray-900/30 hover:border-purple-400/60 hover:bg-gray-800/40"
             }`}
             onClick={() => onRoleSelect("professor")}
           >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)